packages/api-server/src/fastify.ts

```diff
@@ -47,7 +47,7 @@ export function loadFastifyConfig() {
   }
 
   if (!isServerConfigLoaded) {
-    console.log(`Loading server config from ${serverConfigPath} \n`)
+    console.log(`Loading server config from ${serverConfigPath}`)
```

Comment:
Seemed unnecessary and makes a break in the otherwise one-after-another line of logs